edgelet/iotedge/src/main.rs

```diff
@@ -323,6 +355,76 @@ fn run() -> Result<(), Error> {
             tokio_runtime.block_on(Logs::new(id, options, runtime()?).execute())
         }
         ("version", _) => tokio_runtime.block_on(Version::new().execute()),
+        ("cssbundle", Some(args)) => {
+            let location = args.value_of("location").unwrap_or_default();
+            let tail = args
+                .value_of("tail")
+                .and_then(|a| a.parse::<LogTail>().ok())
+                .unwrap_or_default();
+            let since = args
+                .value_of("since")
+                .and_then(|s| parse_since(s).try_into().ok())
+                .unwrap_or_default();
+            let options = LogOptions::new()
+                .with_follow(false)
+                .with_tail(tail)
+                .with_since(since);
+            tokio_runtime.block_on(Bundle::new(options, location, runtime()?).execute())
+        }
         (command, _) => tokio_runtime.block_on(Unknown::new(command.to_string()).execute()),
     }
 }
+
+fn parse_since(since: &str) -> i64 {
```

Comment:
Since the underlying API takes `i32`, this should perform the fallible conversion to `i32` and return that, instead of `i64`